WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -1,5 +1,5 @@
 <table class="table table-striped table-condensed admin-competitions">
-  <% toggle_tbody_message = title.include?("last") ? "hide" : "show" %>
+  <% toggle_tbody_message = title.include?("Past") ? "show" : "hide" %>
```

Comment:
No no, if we are on state = present, only ongoing&upcoming will appear, and it will be visible by default.
This is the toggle_**message**, so 'show' here means the div will be hidden at first.